service-commands/scripts/allClockValues.js

```diff
@@ -0,0 +1,152 @@
+const fs = require('fs')
+
+const { map } = require('lodash')
+const axios = require('axios')
+const retry = require('async-retry')
+async function getUsers(discoveryProvider, offset, limit) {
+  return (
+    await retry(
+      () =>
+        axios({
+          method: 'get',
+          url: '/users',
+          baseURL: discoveryProvider,
+          params: { offset, limit, is_creator: true }
+        }),
+      { retries: 5 }
+    )
+  ).data.data.map(user => ({
+    ...user,
+    creator_node_endpoint: user.creator_node_endpoint
+      ? user.creator_node_endpoint.split(',').filter(endpoint => endpoint)
+      : []
+  }))
+}
+async function getTracks(discoveryProvider, batchSize = 500) {
+  const tracks = []
+
+  let prevBatch = true
+  for (let offset = 0; prevBatch; offset += batchSize) {
+    console.time(`Fetching tracks (${offset} - ${offset + batchSize})`)
+
+    const batch = (
+      await retry(
+        () =>
+          axios({
+            method: 'get',
+            url: '/tracks',
+            baseURL: discoveryProvider,
+            params: { offset, limit: batchSize }
+          }),
+        { retries: 5 }
+      )
+    ).data.data.map(({ metadata_multihash, owner_id }) => ({
+      metadata_multihash,
+      owner_id
+    }))
+    prevBatch = batch.length === batchSize
+    tracks.push(...batch)
```

Comment:
Yes, I changed this so that `trackCids` (formerly `tracks`) is a mapping from all users to all their track CIDs.

However, this still takes only 20 MB of memory, and I think this is reasonable.